toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -598,9 +600,14 @@ func (r *RpmRepoCloner) Close() error {
 // It will gradually enable more repos to consider until the package is found.
 func (r *RpmRepoCloner) clonePackage(baseArgs []string) (preBuilt bool, err error) {
 	const (
-		unresolvedOutputPrefix  = "No package"
-		toyboxConflictsPrefix   = "toybox conflicts"
-		unresolvedOutputPostfix = "available"
+		// With 6 attempts, initial delay of 1 second, and a backoff factor of 3.0 the total time spent retrying will be
+		// 1 + 3 + 9 + 27 + 81 = 121 seconds.
+		//
+		// *NOTE* These values are copied from downloader/downloader.go; they need not be the same but seemed like a
+		// good enough starting point.
```

Comment:
Is this comment here just for the sake of the draft? It looks like something that we may not want in the final version.

Reply:
I'd actually appreciate suggestions of how best to share these -- or perspectives on whether they should be? (Should we create a wrapper with these defaults?)

Perhaps you or @dmcilvaney have thoughts on that?